src/file.cpp

```diff
@@ -62,6 +62,7 @@ POSSIBILITY OF SUCH DAMAGE.
 
 #ifdef TORRENT_DISK_STATS
 #include "libtorrent/io.hpp"
+#include "libtorrent/storage.hpp"
```

Comment:
My idea is have almost zero performance hit if you enable <disk-stats> flag but no enable the actual logging. The silly_hash is called in every open.